services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/permissions/HankeKayttajaService.kt

```diff
@@ -511,6 +538,15 @@ class HankeKayttajaService(
         hankekayttajaRepository.findByHankeIdAndSahkopostiIn(hankeId, emails).map { it.sahkoposti }
 }
 
+data class OwnContactUpdate(@NotBlank val sahkoposti: String, @NotBlank val puhelinnumero: String)
```

Comment:
This is a question of preference, but I find it hard to find the declarations of classes if they're bundled up with services like this. I think it would be reasonable to have the classes for updating the hanke kayttaja and just the contact information in the same file since they're similar enough.